typescript-test-samples/launchdarkly-lambda-dynamodb/metadata.json

```diff
@@ -8,7 +8,7 @@
   "framework": "CDK",
   "services": ["lambda", "dynamodb"],
   "git_repo_url": "https://github.com/aws-samples/serverless-test-samples",
-  "pattern_source": "AWS",
+  "pattern_source": "Customer",
```

Comment:
Could this be 'Partner'?

Reply:
Updated type to Partner and added pattern_source_name as LaunchDarkly